app/src/ui/diff/diff-syntax-mode.ts

```diff
@@ -131,13 +149,17 @@ export class DiffSyntaxMode {
         state.diffLineIndex++
       }
 
-      const token = index ? TokenNames[index] : null
+      if (index === null) {
+        return null
+      }
+
+      const token = TokenNames[index] ?? null
```

Comment:
This is unrelated to the purposes of this PR, but I noticed the `token` ended up being `undefined` instead of `null` sometimes.